### ISSUE TITLE
[feature request]: search only in selected version

### ISSUE THREAD

**Issue by SimonGolms** [opendevstack/ods-documentation#41]
I welcome the implementation of a search bar with PR #29. Unfortunately, it shows all results from all versions. As an end user I would have expected to see only the results from my selected version, which is per default the `latest`. 

![image](https://user-images.githubusercontent.com/17799657/74225919-e1da5900-4cbb-11ea-9e4a-988f757cc4c7.png)

**Comment by tbugfinder** [comment]
https://github.com/opendevstack/ods-documentation/pull/59

**Comment by tbugfinder** [comment]
@michaelsauter just curious - above screenshot includes "Components" at top right. Is this needed as it seems to be lost, too.

**Comment by tbugfinder** [comment]
<img width="492" alt="image" src="https://user-images.githubusercontent.com/15378237/96761375-6c419380-13d9-11eb-8430-8f70e20e02de.png">

**Comment by michaelsauter** [comment]
@tbugfinder No, the "components" bit was due to how the docs were organised prior to 2.x. Back then, all was categorised by repo, which is not ideal as for the user the repo structure doesn't really matter. Now we are free to organise the docs in any structure we think is most helpful.

**Comment by michaelsauter** [comment]
Partially solved by #59. I'd still leave this issue open because a "search in selected version" would be the best default IMHO.

**Comment by tbugfinder** [comment]
I don't think this is feasible right now comparing to antora or couchbase search result pages.
IMHO, grouping would be sufficient.
